projects/dev/src/app/shared/services/api/group.service.ts

```diff
@@ -104,10 +102,10 @@ export class GroupService {
 
   rejectJoinRequest(id, group_ids) {
     return this.http
-      .post(`${this.baseGroupUrl}/${id}/join-requests/reject`, null, {
+      .post<RequestActionResponse>(`${this.baseGroupUrl}/${id}/join-requests/reject`, null, {
         params: {
           group_ids: group_ids.join(","),
-        },
+        }
       })
       .pipe(catchError(this.handleError));
   }
```

Comment:
You fixed the type of the accept but not the reject 😔
Don't you have a error at compilation or in your IDE? I do!

Reply:
It doesn't give compile error actually.
I forgot to add the response to its return type when I specify the return type to the http post method.
My bad!
fixed.